app/controllers/observations_controller/new_and_create.rb

```diff
@@ -194,6 +195,26 @@ def validate_place_name(params)
     success
   end
 
+  def validate_project_checkboxes!
+    return true if params[:project].empty? ||
+                   params[:project][:ignore_proj_dates]
```

Comment:
If `params[:project][:ignore_proj_dates]` is `true` = user has overriden project date restrictions and is adding an out-of-range Observation to a Project.
#### Question
Should the Project admins be notified automatically (or have the option to receive automatic notices)?
IMO yes, but not an immediate priority, and I don't know how this would look as option.
Ideally each admin would have an option for each project, but that's hard.
Easier to have one option that applies to all admins on a project.
Or users could have a user option that applies to all projects that they administer.

Reply:
I'm guessing that 90% of the cases when this happens (if it happens at all) will be a Project admin adding something for some reason.  If some excited user does a bunch of these, I don't think we want to spam the Project admins with a bunch of email.  I think a better solution would be to have a way to indicate and/or search for observations that are outside the official project criteria.

Reply:
I added this to my to-do list.